WooCommerce/Classes/ViewRelated/Orders/Order Creation/CollapsibleProductRowCard.swift

```diff
@@ -47,6 +52,49 @@ struct CollapsibleProductRowCard: View {
                 Text(Localization.priceLabel)
                 CollapsibleProductCardPriceSummary(viewModel: viewModel)
             }
+            .padding(.bottom)
+            HStack {
+                if viewModel.discount == nil {
+                    Button(Localization.addDiscountLabel) {
+                        onAddDiscount()
+                    }
+                    .buttonStyle(PlusButtonStyle())
+                } else {
+                    HStack {
+                        Button(action: {
+                            onAddDiscount()
+                        }, label: {
+                            HStack {
+                                Text(Localization.discountLabel)
+                                Image(uiImage: .pencilImage)
+                                    .resizable()
+                                    .frame(width: Layout.iconSize, height: Layout.iconSize)
+                            }
+                        })
+                        Spacer()
+                        if let discountLabel = viewModel.discountLabel {
+                            Text(minusSign + discountLabel)
+                                .foregroundColor(.green)
```

Comment:
There's probably a ColorStudio color we should be using here instead of the default green

Reply:
There is! Fixed: 69f9ef08340428f135b31128be1f3a8dc6823843